src/core/control/jobs/ImageExport.cpp

```diff
@@ -147,14 +146,17 @@ void ImageExport::exportImagePage(size_t pageId, size_t id, double zoomRatio, Ex
         return;
     }
 
-    if (page->getBackgroundType().isPdfPage() && (exportBackground >= EXPORT_BACKGROUND_UNRULED)) {
+    if (page->getBackgroundType().isPdfPage() && (exportBackground != EXPORT_BACKGROUND_NONE)) {
```

Comment:
Does this change the behavior? Looks like a bug fix.

Reply:
EXPORT_BACKGROUND_UNRULED = 1 and EXPORT_BACKGROUND_NONE = 0, so there is no change in behaviour. I just find this easier to parse.